lua/entities/gmod_wire_expression2/core/glon.lua

```diff
@@ -168,7 +202,7 @@ __e2setcost(10)
 --- Encodes <data> into a string, using [[GLON]].
 e2function string glonEncode(array data)
 	if not glon then
-		error( "Glon is not installed on this server. Please use von instead.", 0 )
+		error( "Glon is not installed on this server. Please use von or json instead.", 0 )
```

Comment:
Is there any reason why we have that text a dozen times?
factor the whole if block out into a "check_glon" function